### PR TITLE
Text inputs - editorial feedback incorporated

### PR DESCRIPTION
**Pull request by chri2547**

### CODE REVIEW

File: _component_design/text-input.md [hunk 1]
```diff
@@ -337,7 +337,7 @@ Consider the following guidelines when you construct a multi-line text area:
 - Text areas scroll vertically when the cursor reaches the bottom of the field.
 - A text area can include an optional draggable resize icon in the lower corner.
 - All text input specifications apply to a text area.
-- Text area states follow the same layout and behaviors as text inputs.
+- Text area states follow the same layout and behaviors as [text input]({{site.baseurl}}/components/text-input.html/#states) states.
```
Comment (nimbinatus) [inline _component_design/text-input.md]: Fix for the link, and recommend putting `states` in the link as it's a specific section.
```suggestion
- Text area states follow the same layout and behaviors as [text input states]({{site.baseurl}}/components/text-input.html#states).
```

File: _component_design/text-input.md [hunk 2]
```diff
@@ -337,7 +337,7 @@ Consider the following guidelines when you construct a multi-line text area:
 - Text areas scroll vertically when the cursor reaches the bottom of the field.
 - A text area can include an optional draggable resize icon in the lower corner.
 - All text input specifications apply to a text area.
-- Text area states follow the same layout and behaviors as text inputs.
+- Text area states follow the same layout and behaviors as [text input states]({{site.baseurl}}/components/text-input.html/#states).
```
Comment (nimbinatus) [inline _component_design/text-input.md]: Still need to fix this

```suggestion
- Text area states follow the same layout and behaviors as [text input states]({{site.baseurl}}/components/text-input.html#states).
```

Comment (chri2547) [reply _component_design/text-input.md]: What is there to fix? Remove the '/'? I have tried without the / and the anchor doesn't work. It does work with the /.

Comment (nimbinatus) [reply _component_design/text-input.md]: Hm, when I test it, it doesn't work with the slash, but does without. Will look after demo

**Comment by chri2547** [comment]
Things that make you go ‘hmmmmmmm…’

File: _component_design/text-input.md (hunk 2 above)
Comment (nimbinatus) [reply _component_design/text-input.md]: So I think you'll have to trust that we have to remove the slash on this one. It's an issue with the site's backend. For a comparison, try the following:
http://helix.rax.io/style/typography.html#link
vs
http://helix.rax.io/style/typography.html/#link
The first works; the second does not.

Comment (chri2547) [reply _component_design/text-input.md]: Ok, will make that change and push! Thanks for researching this one!

**Pull request merged**